### ISSUE TITLE
Critical Error: Cannot run `yarn sst dev` with error `Cannot find package 'stream'`

### ISSUE THREAD

**Issue by Johnrobmiller** [serverless-stack/sst#2471]
Spun up a fresh repo using `yarn create sst project-name`. I ran `yarn`, set up the AWS IAM credentials, and double checked to make sure the config file is correct. However, I was stopped dead in my tracks when running `yarn sst dev` with the error: "Cannot find package 'stream'.

Node version: 14.18.3
OS: Mac (M1 processor)

Full error: 

```
(base) johnm@Johns-MacBook-Pro-2 triniti-sst-dir % yarn sst dev
yarn run v1.22.19
$ /Users/johnm/work/triniti-sst/triniti-sst-dir/node_modules/.bin/sst dev
(node:10961) UnhandledPromiseRejectionWarning: Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'stream' imported from /Users/johnm/work/triniti-sst/triniti-sst-dir/node_modules/sst/stacks/app-metadata.js
    at __node_internal_captureLargerStackTrace (internal/errors.js:412:5)
    at __node_internal_addCodeToName (internal/errors.js:168:9)
    at new NodeError (internal/errors.js:322:7)
    at packageResolve (internal/modules/esm/resolve.js:687:9)
    at moduleResolve (internal/modules/esm/resolve.js:728:18)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:842:11)
    at Loader.resolve (internal/modules/esm/loader.js:89:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:242:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:76:40)
    at link (internal/modules/esm/module_job.js:75:36)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10961) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:10961) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 1.37s.```

**Comment by Johnrobmiller** [comment]
I upgraded my node version to 16.19.0, reinstalled node_modeuls, and that solved the issue.

I don't know whether to close this issue because I don't know if the SST devs want SST to work on node v14.

**Comment by fwang** [comment]
It seems `stream/consumers` is only introduced in Node.js16.

Working on a fix

**Comment by fwang** [comment]
Fixed in 2.0.18
